rx/core/operators/zip.py

```diff
@@ -61,7 +57,8 @@ def on_next(left):
                     right = second[index]
                     index += 1
                     try:
-                        result = result_mapper(left, right)
+                        result = (left, right)
+
                     except Exception as ex:
```

Comment:
Just remove the try/except as well. Not needed anymore.

Reply:
Nice catch, we should do the same for the zip operator.